doc/api/net.md

```diff
@@ -183,15 +283,18 @@ added: v0.1.90
 * `backlog` {Number}
 * `callback` {Function}
 
-Start a local socket server listening for connections on the given `path`.
+The `server.listen()` method instructs the server to begin accepting connections
```

Comment:
This is a (pre-existing) mess, IMO.

All the docs here also need to be in the API above, for when options is passed with a .path, but that would cause unreadable duplication, but ATM they have a non overlapping set of information, which is not good, either.

IMO, I would make the .options variant either specifically reference/link to the docs for the path-based .listen() to describe its `.path` option, or else move almost all the docs here (but not the example) up into the options-taking version of the API, and then document this path-taking version as a variant of the options-taking (which is how its actually implemented). The latter has the advantage of keeping everything centralized, at the price of making the one API really big.

Right now, its neither one approach, or the other.